livre/lib/data/MemoryDataSource.cpp

```diff
@@ -88,21 +88,27 @@ MemoryUnitPtr MemoryDataSource::getData( const LODNode& node )
                             _volumeInfo.getBytesPerVoxel();
     const Identifier nodeID = node.getNodeId().getId();
     const uint8_t* id = reinterpret_cast< const uint8_t* >( &nodeID );
-    const uint8_t fillValue = ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
+    const uint8_t opacity =  ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
```

Comment:
what else is a single-value voxel? The color map translates the opacity into a color - no? Or is there another term for it (like RGB)?